plugin/Gui/AdminPageEstate.php

```diff
@@ -231,9 +232,24 @@ public function preOutput()
 			}
 			return $redirectTo;
 		};
-
 		add_filter('handle_bulk_actions-onoffice_page_onoffice-estates', $pClosureDeleteEstate, 10, 3);
 
+		$pClosureDuplicateEstate = function (string $redirectTo, Table\WP\ListTable $pTable)
+		use ($pDI): string {
+			if (in_array($pTable->current_action(), ['duplicate', 'bulk_duplicate'])) {
+				if (!(isset($_GET['listVewId']))) {
```

Comment:
Noticed this last-minute: the IDs should be a parameter of the function like in line 219.
`check_admin_referer` must be used to check if the action was triggered by someone logged in with an admin account.

Reply:
yes, I will check and fix it.

Reply:
Thanks!